client/test/mockApolloClient.tsx

```diff
@@ -0,0 +1,73 @@
+import { HttpLink, InMemoryCache, ApolloClient } from "apollo-client-preset";
```

Comment:
didn't you tell me that apollo shipped with a package with test utils ? can [this](https://www.apollographql.com/docs/react/development-testing/testing/) be useful ?

Reply:
Haha, I was just looking into this ; I don't think this PR needs tests which involve mocking an apollo client in the end, so I'll just leave things like this for now.

Reply:
alright :+1: